src/webui/www/tstool.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
```

Comment:
this is python3 right?
Please put `#!/usr/bin/env python` so linux user can execute directly.

Reply:
(I haven't check this) and please make sure the file is executable:
https://stackoverflow.com/questions/21691202/how-to-create-file-execute-mode-permissions-in-git-on-windows

Reply:
Ok

Reply:
Ok, changed.